packages/nodes-base/nodes/Airtop/actions/common/fields.ts

```diff
@@ -34,30 +34,20 @@ export const profileNameField: INodeProperties = {
 	name: 'profileName',
 	type: 'string',
 	default: '',
-	description:
-		'The name of the <a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Profile</a> to load or create',
-	hint: 'Name of the <a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Profile</a> to load into the session. Must consist only of alphanumeric characters and hyphen "-"',
+	description: 'The name of the Airtop profile to load or create',
+	hint: '<a href="https://docs.airtop.ai/guides/how-to/saving-a-profile" target="_blank">Learn more</a> about Airtop profiles',
```

Comment:
You'll probably want to do the same changes on line 134 of this file or at least try to keep the hint shorter.
<img width="1723" alt="image" src="https://github.com/user-attachments/assets/25464ebc-0d85-45df-a902-172ce9473fb9" />